src/components/button-toggle/button-toggle-test.stories.tsx

```diff
@@ -29,42 +32,34 @@ export const ButtonToggleStory = ({
       onChange={() => action("onChange")("foo")}
       onFocus={() => action("onFocus")("foo")}
       onBlur={() => action("onBlur")("foo")}
-      children={children || childrenSpecialCharacters}
-      {...args}
-    />
+    >
+      {childrenSpecialCharacters || children}
+    </ButtonToggle>
     <ButtonToggle
       name="new-button-toggle"
       key="button-toggle-2"
       onChange={() => action("onChange")("bar")}
       onFocus={() => action("onFocus")("bar")}
       onBlur={() => action("onBlur")("bar")}
-      children={children || childrenSpecialCharacters}
-      {...args}
-    />
+    >
+      {childrenSpecialCharacters || children}
+    </ButtonToggle>
     <ButtonToggle
       name="new-button-toggle"
       key="button-toggle-3"
       onChange={() => action("onChange")("baz")}
       onFocus={() => action("onFocus")("baz")}
       onBlur={() => action("onBlur")("baz")}
-      children={children || childrenSpecialCharacters}
-      {...args}
-    />
+    >
+      {childrenSpecialCharacters || children}
+    </ButtonToggle>
   </div>
 );
 
-# ToggleButton
-
-### Default
-
-<Canvas>
-  <Story
-    name="default"
-    args={{
-      children: "Option",
-      childrenSpecialCharacters: undefined,
-    }}
-  >
-    {ButtonToggleStory.bind({})}
-  </Story>
-</Canvas>
+Default.story = {
+  name: "default",
+  args: {
+    children: "Options",
+    childrenSpecialCharacters: undefined,
+  },
+};
```

Comment:
```suggestion
Default.storyName = "default";
Default.args = {
    children: "Options",
    childrenSpecialCharacters: undefined,
};
```
Apparently the `storyFunction.story` syntax has been deprecated in Storybook v6: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#hoisted-csf-annotations